configs/ot2_defconfig

```diff
@@ -40,11 +41,10 @@ BR2_PACKAGE_OPENTRONS_THERMOCYCLER_FIRMWARE=y
 BR2_PACKAGE_AVRDUDE=y
 BR2_PACKAGE_BOSSA=y
 BR2_PACKAGE_DBUS_PYTHON=y
-BR2_PACKAGE_RPI_USERLAND=y
+BR2_PACKAGE_I2C_TOOLS=y
 BR2_PACKAGE_UBOOT_TOOLS=y
 BR2_PACKAGE_USBMOUNT=y
 BR2_PACKAGE_PYTHON3=y
-BR2_PACKAGE_PYTHON3_BZIP2=y
```

Comment:
This probably shouldn't get deleted. I don't know why menuconfig keeps doing this!

Reply:
Maybe it's because it's mandatorily included as a dependency of something else?

<img width="929" alt="Screen Shot 2020-10-28 at 9 35 47 PM" src="https://user-images.githubusercontent.com/3236864/97515249-0d46c600-1967-11eb-9a0f-4d29cefd57a9.png">

<img width="928" alt="Screen Shot 2020-10-28 at 9 36 14 PM" src="https://user-images.githubusercontent.com/3236864/97515221-03bd5e00-1967-11eb-9ffe-f1a6fb81356a.png">

I couldn't find the other "deselected" packages in the menuconfig, so I'm not sure if they're all like this.

Regardless, this would be...bad behavior, for a config system. I hope there's a way to change it.